devito/cgen_utils.py

```diff
@@ -174,6 +179,13 @@ def _print_IntDiv(self, expr):
     def _print_Byref(self, expr):
         return "&%s" % expr.name
 
+    def _print_Pow(self, expr):
+        base, exp = expr.as_base_exp()
+        if exp < 0:
+            return '1.0/' + self._print(pow_to_mul(base))
```

Comment:
No. ``pow_to_mul`` should be changed to handle negative exponents and turn the Pow into something else, for example into an `IntDiv`